src/pages/themes/submit.astro

```diff
@@ -0,0 +1,105 @@
+---
+import Button from '../../components/Button.astro'
+import Field from '../../components/forms/Field.astro'
+import ImageInput from '../../components/forms/ImageInput.astro'
+import InputField from '../../components/forms/InputField.astro'
+import Switch from '../../components/forms/Switch.astro'
+import TextAreaField from '../../components/forms/TextAreaField.astro'
+import PageTitle from '../../components/PageTitle.astro'
+import Base from '../../layouts/Base.astro'
+---
+
+<Base title="Submit a theme">
+    <PageTitle>
+        <Fragment slot="title">Submit a theme</Fragment>
+        Thank you for sharing your themes with the Astro community! Choose to make
+        your theme free or paid. All payments go directly to you, we don't handle
+        the sale or take a cut.
+    </PageTitle>
+
+    <section class="max-w-screen-md mx-auto px-4">
+        <h3 class="text-3xl mt-8 mb-4">Best practices</h3>
+        <p class="my-4">What we look for when reviewing submissions:</p>
+        <ul class="my-4 whitespace-normal children:before:content-['✅_']">
+            <li>Your theme uses the latest release of Astro</li>
+            <li>
+                For free themes, source code for the theme is available in a
+                public repository
+            </li>
+            <li>All NPM dependencies used in the theme are public</li>
+            <li>
+                Theme preview image or screenshot included (see below for
+                details)
+            </li>
+        </ul>
+    </section>
+
+    <form
+        method="post"
+        class="max-w-screen-sm flex flex-col items-center gap-4 mx-auto py-8 px-4"
+        data-netlify="true"
```

Comment:
Lets add a custom success page that points to `/themes/submit/success`

https://docs.netlify.com/forms/setup/#success-messages

Reply:
Ah yeah, forgot about that too